crates/rpc_client/src/lib.rs

```diff
@@ -0,0 +1,1019 @@
+use rethnet_eth::{Address, Bytes, H256, U256};
+
+// provide interfaces for all of the client functionality depended on by the existing Hardhat
+// Network logic, specifically
+// packages/hardhat-core/src/internal/hardhat-network/provider/fork/{ForkBlockchain,
+// ForkStateManager}.ts
+// and even more specifically, all of the methods shown in these excerpts:
+//
+//      const remote = await this._jsonRpcClient.getTransactionByHash(
+//    const remote = await this._jsonRpcClient.getTransactionReceipt(
+//      const remoteLogs = await this._jsonRpcClient.getLogs({
+//    const rpcBlock = await this._jsonRpcClient.getBlockByHash(blockHash, true);
+//    const rpcBlock = await this._jsonRpcClient.getBlockByNumber(
+//      const noncePromise = this._jsonRpcClient.getTransactionCount(
+//      const accountData = await this._jsonRpcClient.getAccountData(
+//    const remoteValue = await this._jsonRpcClient.getStorageAt(
+
+mod jsonrpc {
```

Comment:
Could you move this module to a separate file, to reduce clutter in this one file?

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/3383/commits/46b679ee205314d7e3c3e0335fd31e59ebfdcba9